test/units/modules/network/junos/test_junos_config.py

```diff
@@ -101,9 +105,14 @@ def test_junos_config_confirm(self):
         self.assertEqual(kwargs['confirm_timeout'], 40)
 
     def test_junos_config_rollback(self):
-        set_module_args(dict(rollback=10))
+        rollback = 10
```

Comment:
Not sure why you pulled this out

Reply:
rollback value passed as input to the module is used in assert below to check right value is passed to function.

Reply:
...right. Well I feel silly now